tuf/utils/role_sort.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	"strings"
+)
+
+// RoleList is a list of roles
+type RoleList []string
+
+// Len returns the length of the list
+func (r RoleList) Len() int {
+	return len(r)
+}
+
+// Less returns true if the item at i should be sorted
```

Comment:
To be clear, I'm 99% certain you are right and this is pointless, I'm just subtracting a point because I tend to be wrong a lot :D  I would also be happy to add it later instead, since I don't want to block this PR necessarily.  (It'd just be nice to get in before the docker integration)